packages/@expo/prebuild-config/src/importExpoModulesAutolinking.ts

```diff
@@ -24,15 +24,14 @@ function tryRequireExpoModulesAutolinking(projectRoot: string): AutolinkingModul
   const resolveOptions = { paths: [projectRoot] };
 
   try {
-    resolvedAutolinkingPath = require.resolve('expo-modules-autolinking/exports', resolveOptions);
-  } catch {}
-  // Fallback to the older version of expo-modules-autolinking
-  try {
-    resolvedAutolinkingPath = require.resolve(
-      'expo-modules-autolinking/build/autolinking',
-      resolveOptions
-    );
-  } catch {}
```

Comment:
Looks like it's always been using the old path anyway 😅 (the fallback should have been inside `catch {}`)